controllers/suite_test.go

```diff
@@ -25,6 +25,17 @@ const (
 	// ready to process events.  The manager starts asychronously, so without a
 	// wait, events may be missed.
 	managerWaitDuration = 1 * time.Second
+
+	// defaultWorkers is the number of concurrent requests that test controllers
+	// can operate on in paralllel.
+	defaultWorkers = 1
+
+	// defaultSyncDelay sets the sync controller to run immediately.
+	defaultSyncDelay = time.Duration(0)
```

Comment:
It would be interesting to see if this works fine even when running envtest in an existing cluster.
We may have to change this zero delay in the future or if we start observing flaky tests.

Reply:
Increased to 1s, will keep an eye on it.